create.sh

```diff
@@ -8,6 +8,8 @@ TEMPLATES=(
   "nodejs"
   "nodejs-typescript"
   "php"
+  "php-laravel"
+  "php-leaf"
```

Comment:
do these repos exist in DevCycleHQ-Labs yet? I was just trying to run the script but it doesn't fetch the source code